versionedRouter.js

```diff
@@ -443,10 +444,14 @@ if (startDestTransformer) {
               );
             } catch (error) {
               logger.error(error);
+              let status = 400;
               const errorString = error.toString();
+              if (error instanceof RespStatusError) {
+                status = error.statusCode;
```

Comment:
It's better to do this to avoid any developer errors.
```suggestion
                status = error.statusCode || status;
```